inst/@sym/solve.m

```diff
@@ -111,49 +111,67 @@
 
   if (nargout == 0 || nargout == 1)
     cmd = { 'eqs = list(); symbols = list()'
+	    'flag = 0'
             'for arg in _ins:'
             '    if arg.is_Relational:'
             '        eqs.append(arg)'
             '    else:'
             '        symbols.append(arg)'
             '#'
-            'if len(symbols) > 0:'
-            '    d = sp.solve(eqs, symbols, dict=True)'
-            'else:'
-            '    d = sp.solve(eqs, dict=True)'
+	    'try:'
+            '	if len(symbols) > 0:'
+            '		d = sp.solve(eqs, symbols, dict=True)'
+            '	else:'
+            '		d = sp.solve(eqs, dict=True)'
+	    'except Exception as e:'
+	    '	return (str(e), "whatev")'
```

Comment:
Return something like `(False, str(e))`: if `flag` is false then `out` has the error message.